Cython/Utility/CommonStructures.c

```diff
@@ -125,36 +133,3 @@ static PyTypeObject *__Pyx_FetchCommonTypeFromSpec(PyObject *module, PyType_Spec
 }
 #endif
 
-
-/////////////// FetchCommonPointer.proto ///////////////
```

Comment:
I think removing this calls for a separate PR. (Specifically, reverting this PR for whatever potential future reason should not reinstate this code.)